project_scrum/project_scrum.py

```diff
@@ -0,0 +1,500 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api, _
+from datetime import date, timedelta
+import re
+import logging
+try:
+    from bs4 import BeautifulSoup
+except ImportError:
+    logger = logging.getLogger(__name__)
+    logger.warning('bs4  are not available in the sys path')
+
+_logger = logging.getLogger(__name__)
+
+class scrum_sprint(models.Model):
```

Comment:
use [CamelCase](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#odoo-python-classes) for class names, one file per model, and they live in /models